dm/dm/portal/api_test.go

```diff
@@ -155,82 +155,6 @@ func (t *testPortalSuite) initTaskCfg() {
 	}
 }
 
-func (t *testPortalSuite) TestCheck(c *C) {
```

Comment:
portal is an unused module, it's remove in later version

so this pr won't try to fix its ut, just delete it